miscelanea/003-scc-and-caps/README.md

```diff
@@ -0,0 +1,492 @@
+    639 ?        Ss     0:00 /usr/sbin/kadmind -P /var/run/kadmind.pid
+    663 ?        Ss     0:00 /usr/libexec/platform-python -I /usr/libexec/ipa/ipa-custodia /etc/ipa/custodia/custodia.conf
+   1699 ?        Ss     0:00 /usr/sbin/certmonger -S -p /var/run/certmonger.pid -n -d2
+   2456 ?        Z      0:00  \_ [certmonger] <defunct>
+   2798 ?        Ssl    0:18 /usr/lib/jvm/jre-1.8.0-openjdk/bin/java -classpath /usr/share/tomcat/bin/bootstrap.jar:/usr/share/tomcat/bin/tomcat-juli.jar:/usr/share/java/ant.jar:/usr/share/java/ant-launcher.jar:/usr/lib/jvm/java/lib/tools
+   2984 ?        Ssl    0:00 /usr/sbin/gssproxy -D
+   3064 ?        Ss     0:00 /usr/sbin/httpd -DFOREGROUND
+   3070 ?        S      0:00  \_ /usr/sbin/httpd -DFOREGROUND
+   3072 ?        Sl     0:00  \_ (wsgi:kdcproxy) -DFOREGROUND
+   3073 ?        Sl     0:00  \_ (wsgi:kdcproxy) -DFOREGROUND
+   3074 ?        Sl     0:02  \_ (wsgi:ipa)      -DFOREGROUND
+   3075 ?        Sl     0:02  \_ (wsgi:ipa)      -DFOREGROUND
+   3076 ?        Sl     0:02  \_ (wsgi:ipa)      -DFOREGROUND
+   3077 ?        Sl     0:02  \_ (wsgi:ipa)      -DFOREGROUND
+   3078 ?        Sl     0:00  \_ /usr/sbin/httpd -DFOREGROUND
+   3079 ?        Sl     0:00  \_ /usr/sbin/httpd -DFOREGROUND
+   3080 ?        Sl     0:00  \_ /usr/sbin/httpd -DFOREGROUND
+   3629 ?        Sl     0:00  \_ /usr/sbin/httpd -DFOREGROUND
+   3353 ?        Ss     0:00 /usr/sbin/oddjobd -n -p /var/run/oddjobd.pid -t 300
+   3568 ?        Ssl    0:04 /usr/sbin/ns-slapd -D /etc/dirsrv/slapd-APPS-PERMANENT-IDMOCP-LAB-ENG-RDU2-REDHAT-COM -i /run/dirsrv/slapd-APPS-PERMANENT-IDMOCP-LAB-ENG-RDU2-REDHAT-COM.pid
+   3607 ?        Ss     0:00 /usr/sbin/krb5kdc -P /var/run/krb5kdc.pid -w 4
+   3608 ?        S      0:00  \_ /usr/sbin/krb5kdc -P /var/run/krb5kdc.pid -w 4
+   3609 ?        S      0:00  \_ /usr/sbin/krb5kdc -P /var/run/krb5kdc.pid -w 4
+   3610 ?        S      0:00  \_ /usr/sbin/krb5kdc -P /var/run/krb5kdc.pid -w 4
+   3611 ?        S      0:00  \_ /usr/sbin/krb5kdc -P /var/run/krb5kdc.pid -w 4
+   3715 ?        Ss     0:00 /usr/sbin/sssd -i --logger=files
+   3716 ?        S      0:00  \_ /usr/libexec/sssd/sssd_be --domain implicit_files --uid 0 --gid 0 --logger=files
+   3717 ?        S      0:00  \_ /usr/libexec/sssd/sssd_be --domain apps.permanent.idmocp.lab.eng.rdu2.redhat.com --uid 0 --gid 0 --logger=files
+   3719 ?        S      0:00  \_ /usr/libexec/sssd/sssd_nss --uid 0 --gid 0 --logger=files
+   3720 ?        S      0:00  \_ /usr/libexec/sssd/sssd_pam --uid 0 --gid 0 --logger=files
+   3721 ?        S      0:00  \_ /usr/libexec/sssd/sssd_ifp --uid 0 --gid 0 --logger=files
+   3722 ?        S      0:00  \_ /usr/libexec/sssd/sssd_sudo --uid 0 --gid 0 --logger=files
+   3723 ?        S      0:00  \_ /usr/libexec/sssd/sssd_pac --uid 0 --gid 0 --logger=files
+```
+
+This is what I get in the main container, when I run with `exit-on-finished`
+argument:
```

Comment:
This is not looking so good.  FreeIPA itself has not started.  But that is OK, it is not a blocker for this investigation.  It would be good to add a comment that the main container has not started FreeIPA as desired, and the investigation can continue later.

Reply:
Thanks, adding clarification at the document.